pkgs/development/tools/misc/libtree/default.nix

```diff
@@ -13,39 +13,25 @@
 
 stdenv.mkDerivation rec {
   pname = "libtree";
-  version = "2.0.0";
+  version = "3.1.0";
 
   src = fetchFromGitHub {
     owner = "haampie";
     repo = "libtree";
     rev = "v${version}";
-    sha256 = "sha256-j54fUwMkX4x4MwL8gMraguK9GqQRBjCC+W6ojFnQdHQ=";
+    sha256 = "sha256-C5QlQsBL9Als80Tv13ex2XS5Yj50Ht8eDfGYAtnh/HI=";
   };
 
-  patches = [
-    # add missing include
-    # https://github.com/haampie/libtree/pull/42
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Thanks, sorry about that.

Reply:
you didn't push it.

Reply:
Dammit.  Sorry again.  One of these days I will stop using open browser tabs as a to-do list...

#172783